### ISSUE TITLE
[BUG] `prefer-commenting-analyzer-ignores` - incorrectly treats doc comments as regular comments

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#1114]
* **Dart code metrics version:** 5.2.1
* **Dart sdk version:** 2.18.6

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
dart_code_metrics:
    rules:
        - avoid-late-keyword
        - prefer-commenting-analyzer-ignores
```

</details>

**What did you do? Please include the source code example causing the issue.**

```dart
/// (some documentation for the property, nothing related to the ignore comment
//ignore: avoid-late-keyword
late bool foo;
```

**What did you expect to happen?**
error because the doc comment is not a regular comment, and therefore is unlikely to contain an explanation as to why the rule is being ignored

**What actually happened?**
no error

**Are you willing to submit a pull request to fix this bug?**
if it's easy

**Comment by incendial** [comment]
Fixed in 5.3.0 🚀
